xarray/tests/test_combine.py

```diff
@@ -680,11 +680,11 @@ def test_nested_combine_mixed_datasets_arrays(self):
             Dataset({"x": [2, 3]}),
         ]
         with pytest.raises(
-                ValueError,
-                match=r"Can't combine datasets with unnamed arrays."
-            ):
+            ValueError, match=r"Can't combine datasets with unnamed arrays."
```

Comment:
```suggestion
            ValueError, match=r"Can't combine datasets with unnamed dataarrays."
```

Tiny clarification that this means datasets with other xarray.datarrays, not something about the numpy arrays inside the xarray.dataset objects.

Reply:
Good clarification.